src/core/EntrySearcher.cpp

```diff
@@ -181,12 +183,16 @@ bool EntrySearcher::searchEntryImpl(Entry* entry)
             found = entry->attributes()->contains(term.word)
                     && term.regex.match(entry->attributes()->value(term.word)).hasMatch();
             break;
+        case Field::Size: {
+            found = term.regex.match(entrySizeKiB).hasMatch();
```

Comment:
Searching by size is only useful if we have < and > defined. Please remove the searcher entries. The more useful approach for a user is to search by `*` and sort the size column largest to smallest.

Reply:
Removed search by size